build.gradle

```diff
@@ -0,0 +1,292 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+  }
+  dependencies {
+    // check updates with https://plugins.gradle.org/
+    classpath 'io.spring.gradle:dependency-management-plugin:1.0.4.RELEASE'
+    classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.2'
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.4'
+    classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
+    classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.6.2'
+    classpath 'org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.6.2'
+  }
+}
+
+// analyze SonarQube with SonarQube!
+apply plugin: 'org.sonarqube'
```

Comment:
the sections that are restricted to base project should be defined in the same location IMO. I suggest to move this sonarqube section to the bottom of the file with artifactory.